src/sqllogictest/src/runner.rs

```diff
@@ -971,7 +972,11 @@ impl<'a> RunnerInner<'a> {
         let now = SYSTEM_TIME.clone();
         let metrics_registry = MetricsRegistry::new();
 
-        let persist_config = PersistConfig::new(&mz_environmentd::BUILD_INFO, now.clone());
```

Comment:
Wonder if we should rename this to be less appealing? Feels like this constructor should only be used in Persist-specific code now...

Reply:
Good idea! I'll do it in a separate PR

Reply:
Whoops! Forgot to link the PR https://github.com/MaterializeInc/materialize/pull/24667